sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableClientWrapper.java

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Is there a reason moving this class from test of`org.apache.beam.sdk.extensions.sql.meta.provider.bigtable` to main of `org.apache.beam.sdk.io.gcp.bigtable`? I see the class is still only used in the tests of former package.

Reply:
I wanted to keep `BigtableHBaseVeneerSettings` class package private. `BigtableHBaseVeneerSettings` exposes some surfaces from gax and I don't want anyone to call it directly. i'll also need to make a lot of changes in GcpApiSurfaceTest for it to pass if `BigtableHBaseVeneerSettings` is public and marked with `@Internal`. Moving the `BigtableClientWrapper` seems to be an easier change, but let me know if you want to keep the original structure!

Reply:
Sorry still not understand, I mean there is no reference to `BigtableClientWrapper` in `org.apache.beam.sdk.io.gcp.bigtable` package; but only tests of `org.apache.beam.sdk.extensions.sql.meta.provider.bigtable` used this class. Will `BigtableClientWrapper` have something to do with BigtableHBaseVeneerSettings? (Not seen in this PR)

Reply:
Yea, Im a bit confused by this as well. BigtableHBaseVeneerSettings shouldnt be relevant here. You would just need to instantiate BigtableDataClient & BigtableTableAdminClient via their settings